### PR TITLE
feature: add new language (lt-LT)

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+msgstr "Sukurti naują pateiktį"
+
+msgid "Create a test"
+msgstr "Sukurti testą"
+
+#, tao-public
+msgid "Delete"
+msgstr "Ištrinti"
+
+msgid "Deleting of \"%s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: It's missing one double quote?
```suggestion
msgid "Deleting of \"%s\""
```

File: locales/lt-LT/messages.po [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+msgstr "Šalinamas \"%s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Missing one `"`

File: locales/lt-LT/messages.po [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+msgid "Publish \"%s\" to"
+msgstr "Paskelbti \"%s\" į"
+
+msgid "Publish the test"
+msgstr "Paskelbti testą"
+
+msgid "Published on:"
+msgstr ""
+
+msgid "Publishing of \"%s"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Missing closing `"`?

File: locales/lt-LT/messages.po [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+msgstr "\"%s paskelbimas"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Missing closing `"`?

File: locales/lt-LT/messages.po [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+msgstr "Testo paskelbimas nepavyko, nes \"%s\" yra tuščias."
+
+msgid "Test-takers"
+msgstr "Testo laikytojai"
+
+#, tao-public
+msgid "Tests"
+msgstr "Testai"
+
+msgid "The shared filesystem parameter should be a boolean value 'true' or 'false'."
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: Missing slashes before `'`?

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 3 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 5 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: it is not needed here, it is checked

File: locales/lt-LT/messages.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 2 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po (hunk 4 above)
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

File: locales/lt-LT/messages.po [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+msgid "Exported %1$s to %2$s"
+msgstr "Eksportuoti %1$s į %2$s"
+
+msgid "Exporting %s"
+msgstr "Eksportuojama %s"
+
+msgid "Guest access"
+msgstr "Svečio prieiga"
+
+msgid "Importing %1$s files into '%2$s\"
```
[the remark this answers is not in the document]
Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: fixed

**Pull request merged**